event.js

```diff
@@ -0,0 +1,50 @@
+// When you start typing into the navbar's text input, the description
+// of the currently selected car should be bound to what you are typing in
+// and match it exactly.
+
+var carLot = (function(carLot) {
+  var enterText = document.getElementById('addText');
+  var allCards = document.getElementsByClassName('col-xs-4');
+  var changeDesc =document.getElementsByClassName('changeDesc');
+  var carId = ""
+
+  carLot.activateEvents = function () {
+    enterText.addEventListener("keyup", function(){
+      if(enterText.length !== 0){
+        changeDesc[carId].innerHTML = enterText.value;
+      }
+    });
+    for (var a = 0; a < allCards.length; a++) {
+      allCards[a].addEventListener("click", function(event){
+        enterText.focus();
+        enterText.value = "";
+        console.log("event", event);
+        for (var b = 0; b < allCards.length; b++) {
```

Comment:
These reset and style change functions should be in their own iife and augmented so you can call them.